vulture/core.py

```diff
@@ -282,6 +325,13 @@ def get_unused_code(self, min_confidence=0, sort_by_size=False):
         """
         Return ordered list of unused Item objects.
         """
+
+        def _has_noqa(obj):
+            return obj.first_lineno in (
+                self.noqa_matches[ISSUE_CODE_MAP[obj.typ]]
+                or self.noqa_matches["all"]
+            )
```

Comment:
This will fail if, e.g., "noqa V001" is given for line 10, "noqa" is given for line 5 and the object appears on line 5.

Reply:
Oh yes, nice catch. We need a union of two sets here.